docs/styles/style.css

```diff
@@ -0,0 +1,95 @@
+* {
+  margin: 0;
+}
+
+body {
+  font-size: 16px;
+  font-family: "Roboto", Arial, sans-serif;
+  line-height: 1.5; /* baseline height 24p x*/
+}
+
+p {
+  font-size: 1rem;
+  margin-bottom: 1.5rem;
+}
+
+p.lead {
```

Comment:
Works but you could also go with `.lead`. With that you could later decide to change the html to an other tag and you keep the formatting.